changelog.txt

```diff
@@ -1,123 +1,277 @@
-0.4.1
-
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========	
+
+[0.5.0] -- 2019-0x-xx
+---------------------
+Added
++++++
+
+  - Add GETAR file reader/writer.
+  - Rowan is now a dependency.
+	- Add shape_dict representation to Shape classes.
+	- Add support for particle properties: 
+		- mass 
+		- charge 
+		- diameter
+		- image
+		- moment of inertia 
+		- angular momentum
+	- Add support for reading/writing shapes in GSD via hpmc state.
+	- Add universal reader/writer with format detection.
+	- Add orientable attribute to spheres.
+	- Extend list of supported shape classes: 
+		- ellipsoid 
+		- polygons
+		- spheropolygon
+		- convex polyhedron
+		- convex spheropolyhedron
+
+Changed
++++++++
+	- Improve unit test coverage.
+	- Revise documentation.
+	- Move shape definitions to separate module.
+	- Raise AttributeError if accessing an invalid Frame or Trajectory property.
+	- Rename several existing shape classes.
```

Comment:
Make sure to list any backwards-incompatible and user-relevant changes first. Especially backwards-incompatible changes (if present) should be highly visible here. As a user I would expect to be alerted to possible breaking changes as part of this section.

Reply:
So would ordering things like in 7222456 be better?